.eslintrc.js

```diff
@@ -6,22 +6,30 @@ module.exports = {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
     },
+
     {
       files: ['*.js'],
       parserOptions: {
         sourceType: 'script',
       },
       extends: ['@metamask/eslint-config-nodejs'],
     },
+
     {
       files: ['*.test.ts', '*.test.js'],
-      extends: ['@metamask/eslint-config-jest'],
+      extends: [
+        '@metamask/eslint-config-jest',
+        '@metamask/eslint-config-nodejs',
+      ],
     },
   ],
+
   ignorePatterns: [
     '!.eslintrc.js',
     '!.prettierrc.js',
     'dist/',
-    'storybook-static/',
+    '.yarn/',
+    'storybook-static',
+    'docs/utils/getCSSVariablesFromStylesheet.ts', // TODO: Fix the types in this file
```

Comment:
matches module template with inclusion of storybook and this util

https://github.com/MetaMask/metamask-module-template/blob/354995b0a4040cbb74f07b42a0fbc279fb5a1ea0/.eslintrc.js#L30-L34